api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -81,6 +81,63 @@ export function toSARoleAssignmentBody(currentUserId: string, specificAccessData
   };
 }
 
+export function toDenySARoleAssignmentBody(currentUserId: string, allocateRoleData: AllocateRoleData): any {
+  let requestedrole;
+  switch ( allocateRoleData.roleCategory) {
+    case RoleCategory.JUDICIAL:
+      requestedrole = 'specific-access-judiciary';
```

Comment:
Please create public constants for each

Reply:
to being consistent with other parts of this file , followed the same pattern